src/CodeFormatter/App.config

```diff
@@ -1,42 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Are these binding redirects really necessary? Everything works fine for me without them, and I've tried this both with VS2013 and VS2015 CTP.